src/Components/App.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+import Page from './Page';
+
+function App() {
+  const [todo, setTodo] = useState({
+    todoText: '',
+    todos: [],
+  });
+
+  const { todoText, todos } = todo;
+
+  const handleTextChange = (event) => {
+    setTodo({
+      todoText: event.target.value,
+      todos: [...todos],
+    });
+  };
+
+  const handleAddButton = () => {
+    setTodo({
+      todoText: '',
+      todos: [...todos, todoText],
+    });
+  };
+
+  function handleDeleteButton(event) {
+    todos.splice(todos.indexOf(event.target.name), 1);
+    setTodo({
+      todoText,
+      todos,
+    });
+  }
+
+  return (
+    <Page
+      todoText={todoText}
+      todos={todos}
+      textChange={handleTextChange}
+      addButton={handleAddButton}
+      deleteButton={handleDeleteButton}
```

Comment:
```suggestion
      onTextChange={handleTextChange}
      onAddButton={handleAddButton}
      onDeleteButton={handleDeleteButton}
```
핸들러 네이밍 컨벤션 관한 문서 한번 읽어보시면 좋을 것 같습니다~
- https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364